debian/mgmt.service

```diff
@@ -0,0 +1 @@
+../misc/mgmt.service
```

Comment:
Why not? dh_systemd_{enable,start} will automagically install and start all $packagename.service files.